server/public/styles.css

```diff
@@ -47,3 +47,14 @@ td {
   overflow-y: auto;
   height: calc(100vh - 50px);
 }
+
+.single-entry-container {
+  display: flex;
+  justify-content: center;
```

Comment:
Is this necessary? I think it's making your entry details really squished. What if you make your table `100%` the width of its parent element instead.?